pkg/nodeconfig/nodeconfig.go

```diff
@@ -66,6 +66,38 @@ type nodeConfig struct {
 	additionalLabels map[string]string
 }
 
+// ErrWriter is a wrapper to enable error-level logging inside kubectl drainer implementation
+type ErrWriter struct {
+	log logr.Logger
+}
+
+func (ew ErrWriter) Write(p []byte) (n int, err error) {
+	// log error
+	ew.log.Error(err, string(p))
+	return len(p), nil
+}
+
+// OutWriter is a wrapper to enable info-level logging inside kubectl drainer implementation
+type OutWriter struct {
+	log logr.Logger
+}
+
+func (ow OutWriter) Write(p []byte) (n int, err error) {
+	// log info
+	ow.log.Info(string(p))
```

Comment:
Sample output below. There is only a minor cosmetic issue, it adds a [line break](https://github.com/openshift/windows-machine-config-operator/blob/master/vendor/k8s.io/kubectl/pkg/drain/drain.go#L292).

```
	INFO	controllers.configmap	processing	{"instances in": "windows-instances"}
	DEBUG	wc 172.31.251.202	initializing SSH connection
	INFO	nc 172.31.251.202	evicting pod default/win-webserver-wm-6756f7468f-scbm8

	INFO	nc 172.31.251.202	evicting pod default/win-webserver-wm-6756f7468f-zmnc4


```